ui/components/User.js

```diff
@@ -160,6 +161,21 @@ const User = (props) => {
   const { color, iconButtonClassName, avatarClassName, classes } = props;
 
   const open = Boolean(anchorEl);
+
+
+  if ( userData?.status == 'anonymous' ) {
+    const url =
+         `${capabilitiesRegistry?.provider_url}/user/login?anonymousUserID=${userData?.user_id}&redirect=${window.location.pathname}`
```

Comment:
@MUzairS15 why is this parameter explicit and not generic like `userID`? This is asymmetric to the extent that there isn't an `ActiveUserID` and a `PendingUserID`, so what justifies breaking the pattern here of now explicitly exposing user account status intermingled with the user ID? If user account status is needed to be known, should it not be exposed as it own `status` parameter given that it _is_ its own property in the schema? Are we being schema-driven in our API endpoint behavior?